src/instructlab/model/evaluate.py

```diff
@@ -333,6 +333,12 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--background",
+    type=click.BOOL,
```

Comment:
Should this be an is_flag=True?

Reply:
It should be True by default I think so that we suppress stdout/stderr.

Reply:
Agree that we should suppress by default.

If we do pick another name, like "--verbose-serving" or "--enable-serving-output",  it does flip the desired flag default.

Reply:
I think this is sorted out now, resolving. The flag is `enable-serving-output` and default to False :)

Reply:
Actually, I thought this was addressed already but this should really be `is_flag=True` and not `type=click.BOOL`.